src/Feed/MuteList.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from "react";
+
+import { HexKey, TaggedRawEvent, Lists } from "Nostr";
+import EventKind from "Nostr/EventKind";
+import { Subscriptions } from "Nostr/Subscriptions";
+import useSubscription, { NoteStore } from "Feed/Subscription";
+
+export default function useMutedFeed(pubkey: HexKey) {
+    const sub = useMemo(() => {
+        let sub = new Subscriptions();
+        sub.Id = `muted:${pubkey}`;
```

Comment:
You can use only the start of the pubkey here, so the id is a bit snorter

Reply:
i see what you did there ✨